optuna/integration/mlflow.py

```diff
@@ -239,14 +247,37 @@ def _set_tags(self, trial: optuna.trial.FrozenTrial, study: optuna.study.Study)
         # This sets the tags for MLflow.
         mlflow.set_tags(tags)
 
-    def _log_metric(self, value: Union[float, None]) -> None:
-        """Log the trial result as metric to MLflow.
+    def _log_metrics(self, values: List[Union[float, None]]) -> None:
+        """Log the trial results as metrics to MLflow.
 
         Args:
-            value: Result of trial.
+            values: Results of a trial.
         """
-        trial_value = value if value is not None else float("nan")
-        mlflow.log_metric(self._metric_name, trial_value)
+
+        if isinstance(self._metric_name, list):
+            if len(self._metric_name) != len(values):
+                raise RuntimeError(
+                    "Running multi-objective optimization "
+                    "with {} objective values, but {} names specified. "
+                    "Match objective values and names, or use default broadcasting.".format(
+                        len(values), len(self._metric_name)
+                    )
+                )
+
+            else:
+                names = self._metric_name
+
+        else:
+            if len(values) > 1:
+                # broadcast default name for multi-objective optimization
```

Comment:
A nit.
```suggestion
                # Broadcast default name for multi-objective optimization.
```

Reply:
Addressed in 3e6ada476ce08079eb93e812272a6fb4ad1536f9.

Reply:
Thank you.